index.md

```diff
@@ -4,15 +4,11 @@ layout: lesson
 This Library Carpentry lesson introduces people working in library- and information-related roles to working with data in OpenRefine. At the conclusion of the lesson you will understand what the OpenRefine software does and how to use the OpenRefine software to work with data files.
 
 > ## Prerequisites
-> To complete this lesson you will need to install [OpenRefine](http://openrefine.org/download.html) and download the file [doaj-article-sample.csv](https://github.com/LibraryCarpentry/lc-open-refine/raw/gh-pages/data/doaj-article-sample.csv).
->
-> OpenRefine requires one of these web browsers:
-> * Google Chrome
-> * Chromium
-> * Opera
-> * Microsoft Edge
+> To complete this lesson you will need to:
 > 
-> OpenRefine has some issues with Firefox. Internet Explorer is not supported. 
+> 1. Install OpenRefine or use it through a cloud service
+> 1. Download a data file
+> 1. Use a supported browser
```

Comment:
Two "1." in this list

Reply:
Yes, but that's one of the neat Markdown rendering features ;-)

![image](https://user-images.githubusercontent.com/9948149/188484405-491e5577-f34e-47b0-beeb-97efcc584e4b.png)

Reply:
Doh! Of course it is 👍